pennylane/fermi/conversion.py

```diff
@@ -285,7 +288,277 @@ def _(fermi_operator: FermiSentence, n, ps=False, wire_map=None, tol=None):
+    fermi_operator: Union[FermiWord, FermiSentence],
+    n: int,
+    ps: bool = False,
+    wire_map: dict = None,
+    tol: float = None,
+) -> Union[Operator, PauliSentence]:
+    r"""Convert a fermionic operator to a qubit operator using the Bravyi-Kitaev mapping.
+
+    .. note::
+
+        Hamiltonians created with this mapping should be used with operators and states that are
+        compatible with the Bravyi-Kitaev basis.
+
+    In the Bravyi-Kitaev mapping, both occupation number and parity of the orbitals are stored non-locally.
+    In comparison, :func:`~.jordan_wigner` stores the occupation number locally while storing the parity
+    non-locally and vice-versa for :func:`~.parity_transform`. In the Bravyi-Kitaev mapping, the
+    fermionic creation and annihilation operators for even-labelled orbitals are mapped to the Pauli operators as
+
+    .. math::
+
+        \begin{align*}
+           a^{\dagger}_0 &= \frac{1}{2} \left ( X_0  -iY_{0} \right ), \\\\
+           a^{\dagger}_n &= \frac{1}{2} \left ( X_{U(n)} \otimes X_n \otimes Z_{P(n)} -iX_{U(n)} \otimes Y_{n} \otimes Z_{P(n)}\right ), \\\\
+        \end{align*}
+
+    and
+
+    .. math::
+        \begin{align*}
+           a_0 &= \frac{1}{2} \left ( X_0  + iY_{0} \right ), \\\\
+           a_n &= \frac{1}{2} \left ( X_{U(n)} \otimes X_n \otimes Z_{P(n)} +iX_{U(n)} \otimes Y_{n} \otimes Z_{P(n)}\right ). \\\\
+        \end{align*}
+
+    Similarly, the fermionic creation and annihilation operators for odd-labelled orbitals are mapped to the Pauli operators as
+
+    .. math::
+        \begin{align*}
+           a^{\dagger}_n &= \frac{1}{2} \left ( X_{U(n)} \otimes X_n \otimes Z_{P(n)} -iX_{U(n)} \otimes Y_{n} \otimes Z_{R(n)}\right ), \\\\
+        \end{align*}
+
+    and
+
+    .. math::
+        \begin{align*}
+           a_n &= \frac{1}{2} \left ( X_{U(n)} \otimes X_n \otimes Z_{P(n)} +iX_{U(n)} \otimes Y_{n} \otimes Z_{R(n)}\right ), \\\\
+        \end{align*}
+
+    where :math:`X`, :math:`Y`, and :math:`Z` are the Pauli operators, and :math:`U(n)`, :math:`P(n)` and :math:`R(n)`
+    represent the update, parity and remainder sets, respectively [`arXiv:1812.02233 <https://arxiv.org/abs/1812.02233>`_].
+
+    Args:
+        fermi_operator(FermiWord, FermiSentence): the fermionic operator
+        n (int): number of qubits, i.e., spin orbitals in the system
+        ps (bool): whether to return the result as a PauliSentence instead of an
+            Operator. Defaults to False.
+        wire_map (dict): a dictionary defining how to map the orbitals of
+            the Fermi operator to qubit wires. If None, the integers used to
+            order the orbitals will be used as wire labels. Defaults to None.
+        tol (float): tolerance for discarding the imaginary part of the coefficients
+
+    Returns:
+        Union[PauliSentence, Operator]: a linear combination of qubit operators
+
+    **Example**
+
+    >>> w = FermiWord({(0, 0) : '+', (1, 1) : '-'})
```

Comment:
Might be nice to construct this with the `from_string` function or the `Fermi` and `FermiA` operators (not a blocker for this PR)